rinja_derive/src/tests.rs

```diff
@@ -135,6 +112,34 @@ struct Foo {{ {} }}"##,
     }
 }
 
+fn jinja_to_rust(jinja: &str, fields: &[(&str, &str)]) -> syn::Result<syn::File> {
```

Comment:
No no, I meant like https://github.com/rinja-rs/rinja/blob/master/testing/tests/simple.rs#L13-L35. Sorry, should have been more clear. No need to check the generated rust code here.

Reply:
If the same code is generated, then the same code is rendered. It's simpler this way than writing tests with 4 + 4 = 8 cases (all variants at the front and back), or 4 * 4 = 16 tests (cross of all variants).

Reply:
I'm not convinced but there are tests, so all good for me.